clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/TitusCachingProviderConfig.groovy

```diff
@@ -57,20 +59,45 @@ class TitusCachingProviderConfig {
     } as Collection<NetflixTitusCredentials>
     allAccounts.each { NetflixTitusCredentials account ->
       account.regions.each { region ->
-        agents << new TitusClusterCachingAgent(
-          titusCloudProvider,
-          titusClientProvider,
-          account,
-          region,
-          objectMapper,
-          registry,
-          awsLookupUtilProvider,
-          pollIntervalMillis,
-          timeOutMilis
-        )
+        if (!account.splitCachingEnabled) { //default case
+          agents << new TitusClusterCachingAgent(
+            titusCloudProvider,
+            titusClientProvider,
+            account,
+            region,
+            objectMapper,
+            registry,
+            awsLookupUtilProvider,
+            pollIntervalMillis,
+            timeOutMilis
+          )
+        } else { //use new split caching for this whole account
```

Comment:
I've chosen to split the caching by account, not region, because in the controllers account is passed in and not region. I didn't want to support two key types w/in the same account for the transition because it would be a lot of extra code for little benefit.

Reply:
seems reasonable to me